coins/src/scripts/coingecko.ts

```diff
@@ -285,7 +285,7 @@ async function getAndStoreCoins(coins: Coin[], rejected: Coin[]) {
               PK: normalizedPK,
               SK: 0,
               created,
-              decimals,
+              decimals: Number(decimals),
```

Comment:
can you verify if this field exists, and is valid number after this transform before transforming it